tests/test_feedexport.py

```diff
@@ -238,11 +240,19 @@ def run_and_export(self, spider_cls, settings=None):
                 spider_cls.start_urls = [s.url('/')]
                 yield runner.crawl(spider_cls)
 
-            with open(res_name, 'rb') as f:
-                defer.returnValue(f.read())
+            with open(res_path, 'rb') as f:
+                content = f.read()
 
         finally:
-            shutil.rmtree(tmpdir)
+            # FIXME: Windows fails to remove the file because FeedExporter 
+            # keeps a reference to the temporal file even after
+            # the spider finished.
```

Comment:
Should we open an issue for that?

Reply:
+1

Reply:
https://github.com/scrapy/scrapy/issues/3391